frameworks/kafka/docs/configure.md

```diff
@@ -194,6 +195,28 @@ The name of this Kafka instance in DC/OS. This is an option that cannot be chang
 *   **In DC/OS CLI options.json**: `name`: string (default: `kafka`)
 *   **DC/OS web interface**: The service name cannot be changed after the cluster has started.
 
+## Kill Grace Period
+
+The number of seconds to grant each broker to cleanly shutdown in response to SIGTERM.
+The option is termed `brokers.kill_grace_period`.
+
+The Graceful Shutdown feature is especially important for large-scale deployments.
+This configuration option is used to provide the broker sufficient time during
```

Comment:
Use the graceful shutdown configuration option to provide the broker sufficient time during shutdown. This ensures that all in-memory data....